slide-deck.css

```diff
@@ -18,13 +22,20 @@ slide-deck {
 
   &[blank-slide]::after {
     content: '';
-    background-color: var(--blank-color, black);
+    background-color: var(--blank-color);
     position: absolute;
     inset: 0;
+    z-index: -1;
+  }
+
+  &[blank-slide=black] {
+    --blank-color: var(--color-dark);
+    --text-color: var(--color-light);
   }
 
   &[blank-slide=white] {
-    --blank-color: white;
+    --blank-color: var(--color-light);
+    --text-color: var(--color-dark);
```

Comment:
@stacy I think the purpose of the blank slide is hiding what's on the screen - it's a way to temporarily 'pause' the presentation, either with a white or black overlay. Moving the overlay behind the text, and then adding text contrast kinda defeats the purpose, right?

Reply:
@mirisuzanne that was not clear to me at all, and for some reason the "blank" in the name made me think it was more like we are starting with a blank slate, a default slide instead of the more obvious intention of "this should not show anything and it should be blank or empty" 🤦🏻‍♀️ . I don't really know what design issues need to be solved in which PRs/Issues right now, and what is a bug vs what isn't yet ready so I feel like I am overthinking things and not being very helpful yet.

Reply:
That makes sense. I think right now the biggest design issues are the comment above (UX questions about how this should work - what views should be available) and actually designing the slide templates over on the workshop repo. It's ok for design to be pretty minimal in this repo for now.